### PR TITLE
LPD-34944 Duplicate key value in unique index IX_8CCEB8CB at com.liferay.fragment.internal.upgrade.v1_1_0.PortletPreferencesUpgradeProcess

### PR DESCRIPTION
**Pull request by lfbesada**
https://liferay.atlassian.net/browse/LPD-34944

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by lfbesada** [comment]
ci:test:sf

File: modules/apps/fragment/fragment-test/src/testIntegration/java/com/liferay/fragment/internal/upgrade/v1_1_0/test/PortletPreferencesUpgradeProcessTest.java [hunk 1]
```diff
@@ -104,6 +121,110 @@ public void testUpgrade() throws Exception {
...
+				null, TestPropsValues.getUserId(), layout.getGroupId(), 0,
+				fragmentEntry.getFragmentEntryId(), segmentsExperienceId,
+				layout.getPlid(), fragmentEntry.getCss(),
+				fragmentEntry.getHtml(), fragmentEntry.getJs(),
+				fragmentEntry.getConfiguration(), StringPool.BLANK,
+				StringPool.BLANK, 0, StringPool.BLANK, fragmentEntry.getType(),
+				serviceContext),
+			layout, null, 0, segmentsExperienceId);
+
+		ContentLayoutTestUtil.getRenderLayoutHTML(
```
[the remark this answers is not in the document]
Comment (lfbesada) [reply modules/apps/fragment/fragment-test/src/testIntegration/java/com/liferay/fragment/internal/upgrade/v1_1_0/test/PortletPreferencesUpgradeProcessTest.java:326]: It is needed to create the portlet preferences :-/

**Comment by ruben-pulido** [comment]
ci:forward

**Pull request closed without merging**